### PR TITLE
Editorial: Update IEEE 754 standard version

### LINKED ISSUE #1650: A new edition of IEEE-754 has been published
Note that  IEEE 754-2019 has been published.

Changes summarized in http://754r.ucbtest.org/background/ 

Probably nothing that impacts ECMA-262 but we may want to update any normative references.

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Closes #1650

### CODE REVIEW

**Comment by PrimerMediaGroup** [comment]
Rename Spacebookclub.com so we don't lose our profile pictures and information

**Comment by michaelficarra** [comment]
If there's nothing that affects observable behaviour of ECMAScript programs, what's the motivation to update the normative reference? Sell more copies of IEEE-754?

**Comment by apaprocki** [comment]
To the contrary, the observable behavior is the fact that the reference has updated with no change, signaling to readers that the spec is and was compatible with both versions. Otherwise, someone could ask themselves “wait, why the *old* standard?” and spend some amount of time finding the answer.

**Comment by michaelficarra** [comment]
@apaprocki Then, for the same reason, we should mention that either will do.

**Comment by apaprocki** [comment]
What do you think about normatively referencing the 2019 standard, and then providing a "conformance note" (those words) that a 2008 conformant implementation is also acceptable?

Whether or not that extends to say something along the lines of "there are no normative changes between IEEE-754:2008 and IEEE-754:2019 that affect the ECMA-262 specification" is up to the editors.

**Comment by chicoxyzzy** [comment]
Should this note be added to Bibliography?

@ljharb IPR check is failing. Do I need to do something with this?

**Comment by michaelficarra** [comment]
@apaprocki I would strongly prefer the latter wording.

**Comment by ljharb** [comment]
@chicoxyzzy no; it's not a required check. it currently fails on every PR because there's no way to get encrypted credentials safely into that script. I'm working on removing it from PRs (but leaving it on master) until that's solved; ignore it in the meantime.

**Comment by chicoxyzzy** [comment]
@michaelficarra where in spec do you think we should put a note about IEEE 754-2008? It seems to me that the Bibliography section is not the proper place.

**Comment by syg** [comment]
@chicoxyzzy I think a sensible place to put the note that either 2008 or 2019 versions are sufficient is in the bibliography. To make that discoverable, I was thinking existing mentions of IEEE 754 could be linked to jump to the biblio entry. What do you think?

**Comment by chicoxyzzy** [comment]
@syg sounds good. I'll update the PR on weekend.

**Comment by chicoxyzzy** [comment]
- [x] add a note
- [x] add links to bibliography

File: spec.html [hunk 1]
```diff
@@ -43232,8 +43232,11 @@ <h1>Colophon</h1>
 <emu-annex id="sec-bibliography">
   <h1>Bibliography</h1>
   <ol>
-    <li>
-      IEEE Std 754-2008: <i>IEEE Standard for Floating-Point Arithmetic</i>. Institute of Electrical and Electronic Engineers, New York (2008)
+    <li id="sec-bibliography-ieee754">
```
Comment (ljharb) [inline spec.html]: i wonder if instead of an li id, we could have a `<dfn>` around `IEEE Std 754-2019` and then it would get auto-linked?

Comment (chicoxyzzy) [reply spec.html]: I didn't know about `<dfn>`. Fixed!

**Pull request merged**